integration/benchmark/server/control.sh

```diff
@@ -21,7 +21,8 @@ commits_file="/opt/commits.recent"
 load_types=("token-transfer" "create-account" "ledger-heavy" "evm-transfer")
 
 # get the merge commits from the last week from master ordered by author date
-for commit in $(git log  --merges --first-parent  --format="%S:%H" origin/master --since '1 week' --author-date-order )
+# TEMPORARY: DO NOT MERGE!!
```

Comment:
ok to merge?

Reply:
ah thanks for that :) , I forgot about it! 

I wish we had CI for this. Something like this https://github.com/JJ/github-pr-contains-action but listening to "DO NOT MERGE".